c/src/ml-api-service-offloading.c

```diff
@@ -580,29 +611,84 @@ _mlrs_create_edge_handle (ml_service_s * mls, edge_info_s * edge_info)
   return ret;
 }
 
+/**
+ * @brief Set ml service training offloading handle.
+ */
+void
+ml_service_offloading_set_training_handle (ml_service_h handle,
+    void *training_handle)
+{
+  ml_service_s *mls = NULL;
+  _ml_service_offloading_s *offloading_s = NULL;
+
+  if (!handle) {
+    _ml_error_report
+        ("The parameter, 'handle' (ml_service_h), is NULL. It should be a valid ml_service_h.");
+    return;
+  }
+
+  mls = (ml_service_s *) handle;
+
+  offloading_s = (_ml_service_offloading_s *) mls->priv;
+  offloading_s->priv = training_handle;
```

Comment:
Is it ok with no null-checking for ```training_handle```?

Reply:
Yes, 
It is nullable.
It is also used to set null in offloading->priv when destroying the training offloading handle.